dbt/adapters/snowflake/impl.py

```diff
@@ -175,3 +175,6 @@ def standardize_grants_dict(self, grants_table: agate.Table) -> dict:
 
     def timestamp_add_sql(self, add_to: str, number: int = 1, interval: str = "hour") -> str:
         return f"DATEADD({interval}, {number}, {add_to})"
+
+    def valid_incremental_strategies(self):
+        return ["merge", "delete+insert"]
```

Comment:
Let's support `append` explicitly:
```suggestion
        return ["append", "merge", "delete+insert"]
```
in addition to supporting it _implicitly_, when a `unique_key` isn't specified for either `merge` or `delete+insert`